app/src/main/java/io/github/mattpvaughn/chronicle/views/ModalBottomSheetSpeedChooser.kt

```diff
@@ -1,31 +1,29 @@
 package io.github.mattpvaughn.chronicle.views
 
 import android.content.Context
+import android.content.SharedPreferences
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.lifecycle.ViewModelProvider
 import com.google.android.material.bottomsheet.BottomSheetDialogFragment
 import com.google.android.material.chip.Chip
 import com.google.android.material.chip.ChipGroup
 import com.google.android.material.slider.Slider
 import io.github.mattpvaughn.chronicle.application.MainActivity
+import io.github.mattpvaughn.chronicle.data.local.PrefsRepo
 import io.github.mattpvaughn.chronicle.databinding.ModalBottomSheetSpeedChooserBinding
-import io.github.mattpvaughn.chronicle.features.currentlyplaying.CurrentlyPlayingViewModel
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import timber.log.Timber
 import javax.inject.Inject
 
 @ExperimentalCoroutinesApi
 class ModalBottomSheetSpeedChooser : BottomSheetDialogFragment() {
 
-    @Inject
-    lateinit var viewModelFactory: CurrentlyPlayingViewModel.Factory
```

Comment:
I'm pretty sure since `CurrentlyPlayingViewModel.Factory` isn't specifically scoped, the viewModel we injecting here was a different instance than the one in `CurrentlyPlayingViewModelFragment`.

Ended up pulling this out anyways since it's probably better for the two to not be coupled anyways. Now we can add a `ModalBottomSheetSpeedChooser` anywhere in the app without needing a `CurrentlyPlayingViewModel`